src/api/app/controllers/search_controller.rb

```diff
@@ -125,6 +125,9 @@ def owner
       owners = OwnerSearch::Assignee.new(params).for(params[:binary])
     elsif (obj = owner_group_or_user)
       owners = OwnerSearch::Owned.new(params).for(obj)
+    elsif params[:project].blank? && params[:package].present?
+      # package container name without specified project
+      owners = OwnerSearch::Assignee.new(params).for_package(params[:package])
```

Comment:
This is still already happening two lines below in `owner_packages_or_projects`...

https://github.com/openSUSE/open-build-service/blob/6e6f32d5b7a1051136828d3bd9f3f8a4d5851c24/src/api/app/controllers/search_controller.rb#L92-L121

Is there anything missing from that?